output.c

```diff
@@ -63,7 +66,15 @@ void output_log_layers(struct liftoff_output *output) {
 		return;
 	}
 
-	liftoff_log(LIFTOFF_DEBUG, "Layers on CRTC %"PRIu32":", output->crtc_id);
+	liftoff_log(LIFTOFF_DEBUG, "\n== Apply request for output %"PRIu32" ==", output->crtc_id);
```

Comment:
I'd prefer not to add `\n` chars in logging output, because some loggers might add a prefix (with the current time) and this would break it.

Reply:
Ok, makes sense. But then I would try to expand the API as in communicating when a line should be kind of a "header". Otherwise the output is really difficult to read with all the layer and plane information.

Reply:
See latest commit 6c1d2a1 with a proposal for such an API.